internal/schema/v1/search_history.go

```diff
@@ -0,0 +1,30 @@
+package schema
+
+import "github.com/Decentr-net/cerberus/internal/schema/types"
+
+// SearchHistory is user's search history.
+type SearchHistory struct {
+	Timestamp
+
+	Engine     string `json:"searchEngine"`
```

Comment:
Let's rename to engine and query? We're already in the search context.

Reply:
done